doc/api/cli.md

```diff
@@ -771,6 +771,51 @@ Export keyword before a key is ignored:
 export USERNAME="nodejs" # will result in `nodejs` as the value.
 ```
 
+### `--env-file-override-local`
+
+> Stability: 1.1 - Active development
+
+<!-- YAML
+added: REPLACEME
+-->
+
+Override existing environment variables on your machine with values specified in
+your environment files.
+
+```bash
+node --env-file=.env --env-file-override-local index.js
+```
+
+Example:
+
+Assume you have a local environment variable on your machine
+
+```bash
+API_KEY="local-key"
+```
+
+Meanwhile, your `.env` file contains:
+
+```bash
+API_KEY='env-custom-key'
+```
+
+If you want to use the value from the `.env` file, you should add the
+`--env-file-override-local` flag when running your application.
+This flag requires the use of the `--env-file` flag to load environment files.
+
+```js
+console.log(process.env.API_KEY); // 'env-custom-key'
+```
+
+If you prefer to keep the local value, do not add the
+`--env-file-override-local` flag or avoid providing the override option to
+[`process.loadEnvFile()`][]
+
+```js
+console.log(process.env.API_KEY); // 'local-key'
+```
```

Comment:
I think the example should go more straight to the point
````suggestion
```console
$ echo 'API_KEY="local-key"' > .env
$ export API_KEY='env-custom-key'
$ node --env-file=.env --env-file-override-local -p 'process.env.API_KEY'
local-key
$ node --env-file=.env -p 'process.env.API_KEY'
env-custom-key
```
````